src/map/Map.js

```diff
@@ -696,7 +696,8 @@ L.Map = L.Evented.extend({
 		for (var i = 0; i < targets.length; i++) {
 			if (targets[i].listens(type, true)) {
 				targets[i].fire(type, data, true);
-				if (data.originalEvent._stopped) { return; }
+				if (data.originalEvent._stopped
+					|| (targets[i].options.nonBubblingEvents && targets[i].options.nonBubblingEvents.indexOf(type) !== -1)) { return; }

```

Comment:
It's fine to keep only for arrays since string indexOf doesn't need an util method. It's an internal utility after all, we don't have to make it generic.